cli/container.go

```diff
@@ -172,6 +173,11 @@ func (c *container) config() (*types.ContainerCreateConfig, error) {
 		return nil, err
 	}
 
+	shmSize, err := opts.ParseShmSize(c.shmSize)
```

Comment:
we are missing the `opts.ParseShmSize` function 😄

Reply:
Oh, yes